src/test/java/com/logigear/RegisterTest.java

```diff
@@ -0,0 +1,31 @@
+package com.logigear;
+
+import helper.DataHelper;
+import helper.LoggerHelper;
+import models.Account;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.HomePage;
+import page_objects.RegisterPage;
+
+public class RegisterTest extends BaseTest {
+
+    private final HomePage homePage = new HomePage();
+    private final RegisterPage registerPage = new RegisterPage();
+
+
+    @Test(description = "TC07 - User can create new account")
+    public void TC07() {
+        LoggerHelper.startTestCase("TC07 - User can create new account");
+        homePage.goToRegisterPage();
+
+        String password = DataHelper.getRandomValidPassword();
+        Account account = new Account(DataHelper.getRandomValidEmail(),
+                password, DataHelper.getRandomValidPid());
+        registerPage.register(account);
```

Comment:
why did you create variable for password only? Should be consistent, create variables for all the data or neither all.

Reply:
Thank you for the review, I will edit it now